lua/entities/hl_camera_ghost/cl_init.lua

```diff
@@ -0,0 +1,10 @@
+include("shared.lua")
+
+function ENT:DrawTranslucent()
+	local camera = self:GetParent()
+	if not camera then return end
+	if not camera:ShouldDraw() then return end
+
+	-- in all other cases, draw
+	self:DrawModel()
+end
```

Comment:
Excellent thought! However I don't think this works if the camera is outside the client's PVS (while the ghost is visible). Need to test.

Reply:
Some googling found me this: https://developer.valvesoftware.com/wiki/Networking_Entities#Transmission_Filters "Note: 	If an entity transmits, it will force all of its parents to transmit as well."

Meaning this is safe.